src/components/FilterList/FilterLIst.tsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+
+import cn from 'classnames';
+
+type Props = {
+  itemCount: number,
+};
+
+export const FilterList: React.FC<Props> = ({ itemCount }) => {
+  const [filterBy, setFilterBy] = useState('All');
+
+  const changeFilter = (newValue: string) => {
+    setFilterBy(newValue);
+  };
+
+  return (
+    <>
+      <span className="todo-count" data-cy="todosCounter">
+        {`${itemCount} `}
+        items left
```

Comment:
```suggestion
        {`${itemCount} items left`}
```
I think interpolation would look cleaner here